kubetest/main.go

```diff
@@ -459,7 +487,7 @@ func prepare() error {
 		return err
 	}
 
-	if err := os.MkdirAll("./_artifacts", 0777); err != nil { // Create artifacts
+	if err := os.MkdirAll(artifacts, 0777); err != nil { // Create artifacts
```

Comment:
I would expect it to be `_artifacts` everywhere on the local disk and `artifacts` on gcs.

Both dockerized-e2e-runner.sh and scenarios/kubernetes_e2e.py use `_artifacts`:
https://github.com/kubernetes/test-infra/blob/master/jenkins/dockerized-e2e-runner.sh#L82
https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L159

bootstrap.py then uploads `_artifacts` to an `artifacts` sub-path in the build dir. This is true for both CI and PR runs:
https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L564
https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L523

Can you find any examples of it being artifacts on the host and/or inside the container?

I suspect the reason we call it _artifacts is because we configure .gitignore to ignore this file when determining whether the version being test is "dirty": https://github.com/kubernetes/test-infra/blob/a5c336f3e6af1929667913377935bf02ea07af60/.gitignore#L7